### ISSUE TITLE
Homebrew Release

### ISSUE THREAD

**Issue by storopoli** [atheriel/xrprof#12]
Dear @atheriel I was the one who asked about Homebrew in rstudio::conf. I have a Mac (Intel) and I could help you to test out `xrprof`. What can I do?

**Comment by atheriel** [comment]
I will file a separate issue explaining how to port this to macOS when possible. Until it has been ported there's not much to do.

Once `xrprof` works on macOS getting it into Homebrew [seems straightforward](https://docs.brew.sh/Formula-Cookbook).

**Comment by storopoli** [comment]
Ok. tag me on the new issue and I will be glad to help!

**Comment by atheriel** [comment]
Blocked by #13.
